### ISSUE TITLE
Code Generation for Swift is not compatible with Swift 3. 

### ISSUE THREAD

**Issue by grantkemp** [a85/POSTMan-Chrome-Extension#2415]
1. Postman Version:Version 4.8.0 (4.8.0)
2. App (Chrome app or Mac app): Mac App 
3. OS details: Latest OSX
4. Is the Interceptor on and enabled in the app: ??
5. Did you encounter this recently, or has this bug always been there: recently since new version was released
6. Expected behaviour: Currently the Swift code thats generates is using an old version. Rename the existing impementation for Swift 2.3 and add a new entry for Swift 3.0
7. Console logs (http://blog.getpostman.com/2014/01/27/enabling-chrome-developer-tools-inside-postman/ for the Chrome App, View->Toggle Dev Tools for the Mac app):
8. Screenshots (if applicable)

![image](https://cloud.githubusercontent.com/assets/751115/19438986/3f5c05fa-9474-11e6-93c3-959123960964.png)

**Comment by vegetableman** [comment]
Thanks @grantkemp for posting this issue 👍 . Will keep you updated on this thread.

**Comment by sdnts** [comment]
@grantkemp Can you try updating to our latest version (4.9) and check if the generated code is correct?

**Comment by grantkemp** [comment]
@madebysid  cheers! will download and test it.  If its good will close this off.  

Great work you guys are doing - I have got loads of my colleagues using it.

**Comment by grantkemp** [comment]
Just tried 4.9.0 and it looks like its still using the old code under Swift (NSURL) - am I looking in the wrong place?
![image](https://cloud.githubusercontent.com/assets/751115/20837193/e3162c4e-b89a-11e6-9c8f-a5a2034dbe1f.png)

(below looks like Swift 1 code!)
```let session = NSURLSession.sharedSession()
let dataTask = session.dataTaskWithRequest(request, completionHandler: { (data, response, error) -> Void in
  if (error != nil) {
    println(error)
  } else {
    let httpResponse = response as? NSHTTPURLResponse
    println(httpResponse)
  }
})

dataTask.resume()```

**Comment by vegetableman** [comment]
@grantkemp yep. Apparently, we missed adding the patch to the final build. Will get back to you on this.

**Comment by grantkemp** [comment]
No worries-looks like a pretty full release anyway - no rush.

**Comment by vegetableman** [comment]
@grantkemp The issue has been fixed in the latest release of `4.9.2` ^_^. Closing this, feel free to reopen if any issues.

**Comment by grantkemp** [comment]
Thanks @vegetableman
